parts/inc/misc

```diff
@@ -227,7 +227,10 @@ __UNDEFINED__ SvRXOK(sv) (!!SvRX(sv))
 #endif
 
 __UNDEFINED__  NOOP          /*EMPTY*/(void)0
-__UNDEFINED__  dNOOP         extern int /*@unused@*/ Perl___notused PERL_UNUSED_DECL
+
+#if { VERSION < 5.27.7 }
+__UNDEFINED__ dNOOP struct Perl___notused_struct
```

Comment:
If you want to override the pre 5.27.7 definition, you need a #undef dNOOP between lines 231 and 232
Otherwise LGTM